### PR TITLE
Feature/update notifications

### PR DESCRIPTION
**Pull request by henriquekano**
### Descrição

Adiciona notificações de novas versõe no admin

### Testes Realizados

Teste unitário do helper que determina a última versão a partir da resposta da api do github (https://developer.github.com/v3/repos/releases/#list-releases-for-a-repository)

### CODE REVIEW

File: tests/app/code/community/Pagarme/Notifications/Helper/Pagarme_Noifications_Helper_Github_Data.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+<?php
+
+class Pagarme_Notifications_Helper_Github_Data extends \PHPUnit_Framework_TestCase
```
Comment (lflimeira) [inline tests/app/code/community/Pagarme/Notifications/Helper/Pagarme_Noifications_Helper_Github_Data.php]: @henriquekano It should have test in the class name by convention.

File: tests/app/code/community/Pagarme/Notifications/Helper/Pagarme_Noifications_Helper_Github_Data.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+{
+    private $helper;
+
+    protected function setUp()
+    {
+        $this->helper = new Pagarme_Notifications_Helper_Data();
+    }
+
+    public function testTest()
```
Comment (lflimeira) [inline tests/app/code/community/Pagarme/Notifications/Helper/Pagarme_Noifications_Helper_Github_Data.php]: Wouldn't be better name this test of testReleaseMessage() or something like that?

File: app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                curl_close($ch);
+
+                if ($statusCode >= 400) {
+                    throw new \Exception();
+                }
+                return $response;
+            });
+            $version = Mage::getConfig()->getNode()->modules->Inovarti_Pagarme->version;
+            if ($this->isLatestVersion($latestVersion['name'], $version)) {
+                return 'Yay. New version!: ' . $latestVersion['html_url'];
```
Comment (leonampd) [inline app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php]: I think we should use the translate methods to show this message.

File: app/code/community/Pagarme/Notifications/Helper/Data.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+class Pagarme_Notifications_Helper_Data extends Mage_Core_Helper_Abstract
+{
+    public function getLatestVersionFor($baseBranch, $fetchVersions)
+    {
+        $resp = $fetchVersions();
```
Comment (leonampd) [inline app/code/community/Pagarme/Notifications/Helper/Data.php]: Looking at the `Inovarti_Pagarme_Block_Adminhtml_Notifications` I don't see any dynamic data. So, we can transform this variable function as a `Inovarti_Pagarme_Block_Adminhtml_Notifications`'s method?

File: app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        ];
+        $ch = curl_init();
+        curl_setopt($ch,  $defaults);
+        $rawResponse = curl_exec($ch);
+        $response =  json_decode($rawResponse);
+        $statusCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+        curl_close($ch);
+
+        if ($statusCode >= 400) {
+            throw new \Exception();
```
Comment (leonampd) [inline app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php]: Why this exception is empty? I think we should have a message here to explain the error.

Comment (henriquekano) [reply app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php]: Is that really necessary? I mean, I just throw the exception so I can catch it outside. In this case, I just handle it by returning an empty string.

Comment (leonampd) [reply app/code/community/Inovarti/Pagarme/Block/Adminhtml/Notifications.php]: IMHO it's a good practice add a message.

**Pull request merged**